### PR TITLE
Update lockfile

### PR DESCRIPTION
**Pull request by theHamsta**
Account for https://github.com/tree-sitter/tree-sitter-ruby/pull/152

### CODE REVIEW

**Comment by theHamsta** [comment]
@TravonteD

File: lockfile.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "revision": "a22fa5e19bae50098e2252ea96cba3aba43f4c58"
   },
   "teal": {
-    "revision": "6142154f8d78dbf00b9c0e70bf2f994d97e76582"
+    "revision": "cec82d45e630005346d8e9df2414afedd212362f"
```
Comment (vigoux) [inline lockfile.json:114]: We don't want this teal update here.
We'll let CI update that one.

**Review by vigoux — CHANGES_REQUESTED** [review]

File: lockfile.json (hunk 1 above)
Comment (theHamsta) [reply lockfile.json:114]: That is the commit of the CI

Comment (vigoux) [reply lockfile.json:114]: Sorry, did not saw that !

**Review by vigoux — APPROVED** [review]

**Pull request merged**